Scripts/script-FindSimilarIncidents.yml

```diff
@@ -0,0 +1,329 @@
+      hr_result = map(lambda row: dict((camel_case_to_space(k), v) for k, v in row.items()), duplicate_incidents_rows)
+      markdown_result = tableToMarkdown("Duplicate incidents",
+                                        hr_result,
+                                        headers=['Id', 'Name', 'Closed time', 'Occurred time'])
+      demisto.results({'ContentsFormat': formats['markdown'],
+                       'Type': entryTypes['note'],
+                       'Contents': markdown_result,
+                       'EntryContext': context})
+  else:
+      demisto.results('No duplicate incidents has been found')
+type: python
+tags:
+- dedup
+comment: |-
+  Find similar incidents by common incident keys, labels, custom fields or context keys.
+  It's highly recommended to use incident keys if possible (e.g. "type" for same incident type).
+  It's also recommended to avoid using context keys if possible (for example, if the value also appear in label key, use label).
+enabled: true
+args:
+- name: similarIncidentKeys
+  description: Identical incident keys. Comma separated value
+- name: similarLabelsKeys
+  description: 'Similar label keys. Comma separated value. It''s also possible to
+    let X different words between labels, within the following way: label_name:X,
+    where X is the number of words.'
+- name: similarContextKeys
+  description: 'Similar context keys. Comma separated value. It''s also possible to
+    let X different words between labels, within the following way: label_name:X,
+    where X is the number of words.'
+- name: similarCustomFields
+  description: 'Similar custom fields keys. Comma separated value. It''s also possible
+    to let X different words between labels, within the following way: label_name:X,
+    where X is the number of words.'
+- name: hoursBack
+  description: Maximum number of hours for incidents to lookup.
+  defaultValue: "24"
+- name: ignoreClosedIncidents
+  auto: PREDEFINED
+  predefined:
+  - "yes"
+  - "no"
+  description: Ignore close incidents as duplicate candidates.
+  defaultValue: "yes"
+- name: maxNumberOfIncidents
+  description: Maximum number of incidents to query.
+  defaultValue: "1000"
+- name: ignoreMissingKeys
+  auto: PREDEFINED
+  predefined:
+  - "yes"
+  - "no"
+  description: Ignore missing labels\context\incident keys.
+  defaultValue: "no"
+- name: minimumNumberOfKeysToCompare
+  description: The minimum number of keys (context\labels\custom fields\incident)
+    to compare. The default is 1.
+  defaultValue: "1"
+outputs:
+- contextPath: similarIncident.rawId
+  description: Similar incident ID.
+  type: string
+- contextPath: isSimilarIncidentFound
+  description: Is similar incident found? (true\false)
+  type: boolean
+- contextPath: similarIncident
+  description: Similar incident.
+  type: unknown
+- contextPath: similarIncident.name
+  description: Similar incident name.
+  type: string
+scripttarget: 0
+timeout: 300ns
+runonce: false
+releaseNotes: "Add script"
```

Comment:
make releaseNotes a bit more informative, maybe:
"Add automation to find duplicate incident"